.eslintrc

```diff
@@ -69,8 +69,10 @@
         "eslint-plugin/meta-property-ordering": "error",
         "eslint-plugin/no-deprecated-context-methods": "error",
         "eslint-plugin/no-deprecated-report-api": "off",
+        "eslint-plugin/prefer-output-null": "off",
```

Comment:
Can't enable [eslint-plugin/prefer-output-null](https://github.com/eslint-community/eslint-plugin-eslint-plugin/blob/main/docs/rules/prefer-output-null.md) yet since one of the older versions of ESLint that we test against doesn't. support `output: null`.